AutomatticTracks/src/main/java/com/automattic/android/tracks/CrashLogging/CrashLogging.java

```diff
@@ -0,0 +1,138 @@
+package com.automattic.android.tracks.CrashLogging;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.automattic.android.tracks.BuildConfig;
+import com.automattic.android.tracks.TracksUser;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Locale;
+import java.util.Map;
+
+import io.sentry.Sentry;
+import io.sentry.SentryClient;
+import io.sentry.android.AndroidSentryClientFactory;
+import io.sentry.connection.EventSendCallback;
+import io.sentry.event.Event;
+import io.sentry.event.EventBuilder;
+import io.sentry.event.UserBuilder;
+import io.sentry.event.helper.ShouldSendEventCallback;
+import io.sentry.event.interfaces.ExceptionInterface;
+
+public class CrashLogging {
```

Comment:
In the Woo implementation, we created a `CrashUtils` class which just passes its method contents like `log` and `logException` back to `CrashLogging`. I'm concerned that there would be some confusion about which class to use. Also, if someone accidentally uses `CrashLogging.logException()`, it will still work but without the custom logic implemented in `CrashUtils.logException()`. 

```
// this?
CrashLogging.logException()

// or this?
CrashUtils.logException()
```
My suggestion is to make `CrashLogging` not contain static methods at all and require it to be instantiated. Then, `CrashUtils` can instantiate it and configure it itself. 

```kt
object CrashUtils {
    private loggingService: CrashLogging
    fun initCrashLogging() {
        loggingService = CrashLogging()
        ...
    }
}
```

In this way, it will be a bit difficult to accidentally use `CrashLogging` itself and we would naturally rely on `CrashUtils` which has `static` `log()` and `logException()` methods. We can then safely add app-specific logic to `CrashUtils` without policing everyone to use it instead of `CrashLogging`. 

What do you think?